src/cow-react/modules/swap/pure/Row/RowSlippageContent/index.cosmos.tsx

```diff
@@ -1,34 +1,14 @@
-import { CurrencyAmount, Percent, Price, TradeType } from '@uniswap/sdk-core'
+import { Percent } from '@uniswap/sdk-core'
 import { RowSlippageContent, RowSlippageContentProps } from '@cow/modules/swap/pure/Row/RowSlippageContent'
 import { RowSlippageProps } from '@cow/modules/swap/containers/RowSlippage'
-import { SupportedChainId } from '@cowprotocol/cow-sdk'
```

Comment:
removed the trade prop from RowSlippage and just pass native currency directly as that was all that was needed